src/main/java/gel/Storage.java

```diff
@@ -0,0 +1,119 @@
+package gel;
+
+
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.List;
+import java.util.Scanner;
+
+import gel.task.Deadline;
+import gel.task.Event;
+import gel.task.Task;
+import gel.task.Todo;
+
+/**
+ * Storage deals with loading tasks from the file and saving tasks in the file.
+ */
+public class Storage {
+
+    private String filePath;
+    private String directoryPath;
+
+    /**
+     * Constructor of Storage class.
+     *
+     * @param filePath ensures that tasks are stored in the file path.
+     */
+    public Storage(String filePath) {
+        int fileIndex = filePath.lastIndexOf("/");
+        this.directoryPath = filePath.substring(0, fileIndex);
```

Comment:
Should you remove the this keyword? The [java coding standard documentation](https://se-education.org/guides/conventions/java/index.html#variables) mentions to use this keyword only when a field is shadowed by a method or constructor parameter.